### ISSUE TITLE
Add 2FA prompts after initial signin rather than outright, add Authy icon.

### ISSUE THREAD

**Issue by starfoxdot64** [filecoffee/issues#5]
**Is your feature request related to a problem? Please describe.**
As far as signin goes, I can see it becoming a hassle for end users when it comes to 2FA and how it's placed during signin. It'll create questions like "Do I need this?" or "Why is this here?". As far as Authy goes, the default for file.coffee is... comical at best. Here's what Authy thinks file.coffee is:
[image](https://user-images.githubusercontent.com/46762446/117336285-e1922480-ae50-11eb-95f5-ef8812dd8c01.png)

**Describe the solution you'd like**
I'm hoping for an "if this, then that" setup for the 2FA system. If the password is correct, a second screen should appear asking for the 2FA code, like other 2FA-compliant sites (think Google, Apple, Amazon, Facebook, Telegram (albeit in reverse), etc. I'm sure there's a way to contact Authy about getting an icon submitted, but I'm certain that a stock photo of a coffee icon isn't this.

**Comment by abcdan** [comment]
Yes, but it makes it impossible for an attacker to know if you are using 2FA or not. The login endpoint gives the same error for "no account found" and "no 2fa provided" (when you have it enabled).

It makes it a little bit more secure for you. I might make it a bit more clear what it is with a hover message.

**Comment by lukasvdberk** [comment]
I agree with @starfoxdot64  that a second prompt for 2FA is a better option in my opinion

**Comment by abcdan** [comment]
And ideas how I can make it cleaner without having to expose if someone is using it or not?

**Comment by lukasvdberk** [comment]
You can redirect them to a 2FA page after having entered a correct email and password.

**Comment by lukasvdberk** [comment]
What is the current status on this issue?

**Comment by abcdan** [comment]
Still brainstorming on how to make it better.
